src/Analyzers/CSharp/Tests/ConvertNamespace/ConvertToFileScopedNamespaceAnalyzerTests.cs

```diff
@@ -808,8 +805,8 @@ public async Task TestConvertToFileScopedSingleLineNamespace2()
 ",
                 FixedCode = @"
 namespace $$N;
-
-class C { } ",
+class C { } 
```

Comment:
improvement.  the original class was directly underneath the namespace.  the new code preserves that.